### PR TITLE
sync with english version for <position> data type

### PR DESCRIPTION
**Pull request by yin1999**
### Description

sync with english version for <position> data type

### Related issues and pull requests

~blocked by: mdn/content#27945.~

### CODE REVIEW

File: files/zh-cn/web/css/position_value/index.md [hunk 1]
```diff
@@ -5,44 +5,69 @@ slug: Web/CSS/position_value
...
-<length> or <percentage> /* The position on the x-axis, 50% for the y-axis */
+## 插值
 
-/* 2-value syntax */
-keyword keyword          /* A keyword for each direction, the order is irrelevant */
-keyword value            /* The value is the offset for the edge defined by the keyword */
-```
+横坐标值和纵坐标值都独立进行插值。而由于两者插值速度都由同一[缓动函数](/zh-CN/docs/Web/CSS/easing-function)定义，点将沿一条直线移动。
+
+### 形式语法
```
Comment (jasonren0403) [inline files/zh-cn/web/css/position_value/index.md:45]:
```suggestion
## 形式语法
```

File: files/zh-cn/web/css/position_value/index.md [hunk 2]
```diff
@@ -5,44 +5,69 @@ slug: Web/CSS/position_value
...
+
+{{csssyntax}}
+
+## 示例
 
-### 正式语法
+### 有效位置
```
Comment (jasonren0403) [inline files/zh-cn/web/css/position_value/index.md:51]: 或者用“position 值” / “位置值”？ 🤔

**Review by jasonren0403 — COMMENTED** [review]
some suggestions

File: files/zh-cn/web/css/css_values_and_units/index.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
 ---
-title: CSS values and units
+title: CSS 值与单位
 slug: Web/CSS/CSS_Values_and_Units
 ---
 
 {{CSSRef}}
 
-所有的 CSS 声明都包括一个 属性 / 值 对。由于属性不同，对应的值可能是一个单个整数或关键字，也可能是一串包含或不包含计数单位的数字和关键字的集合。CSS 中属性接受一个共同的数据类型（属性的值和对应的单位）。下面是大多数这些数据类型的概览。想要获知每一种数据类型的更详细信息，请分别查看它们的链接。
+所有的 CSS 声明都包括一个“属性/值”对。由于属性不同，对应的值可能是一个单个整数或关键字，也可能是一串关键字和包含或不包含单位的值的集合。CSS 中属性接受一组共同的数据类型（属性的值和对应的单位）。下面是大多数这些数据类型的概览。想要获知每一种数据类型的更详细信息，请分别查看它们的链接。
```
[the remark this answers is not in the document]
Comment (yin1999) [reply files/zh-cn/web/css/css_values_and_units/index.md:8]: As this is the description of the `value` of a CSS attr, maybe using "串" is better.

```css
p {
   margin: 0 0 2px 0;
}

File: files/zh-cn/web/css/position_value/index.md [hunk 4]
```diff
@@ -5,44 +5,69 @@ slug: Web/CSS/position_value
 
 {{CSSRef}}
 
-## 概要
+[CSS](/zh-CN/docs/Web/CSS) `<position>`（或 **`<bg-position>`**）[数据类型](/zh-CN/docs/Web/CSS/CSS_Types)表示用于设置相对于元素盒子的位置的 2 维空间中的坐标。
```
[the remark this answers is not in the document]
Comment (yin1999) [reply files/zh-cn/web/css/position_value/index.md:8]:
```suggestion
[CSS](/zh-CN/docs/Web/CSS) `<position>`（或 **`<bg-position>`**）[数据类型](/zh-CN/docs/Web/CSS/CSS_Types)表示用于设置相对于元素盒子的位置的 2 维空间中的坐标。它被用于 {{cssxref("background-position")}} 和 {{cssxref("offset-anchor")}} 属性。
```

File: files/zh-cn/web/css/position_value/index.md (hunk 2 above)
Comment (yin1999) [reply files/zh-cn/web/css/position_value/index.md:51]:
```suggestion
### 有效位置值
```

File: files/zh-cn/web/css/position_value/index.md [hunk 5]
```diff
@@ -5,44 +5,69 @@ slug: Web/CSS/position_value
...
+10% 20%
+8rem 14px
 ```
-[ [ left | center | right | top | bottom | <percentage> | <length> ] |
-                  [ left | center | right | <percentage> | <length> ] [ top | center | bottom | <percentage> | <length> ] |
-                  [ center | [ left | right ] [ <percentage> | <length> ]? ] &&
-                  [ center | [ top | bottom ] [ <percentage> | <length> ]? ]
-                ]
+
+### 无效位置
```
Comment (yin1999) [inline files/zh-cn/web/css/position_value/index.md:65]:
```suggestion
### 无效位置值
```

**Review by jasonren0403 — APPROVED** [review]
Thanks @yin1999

**Pull request merged**